djconnectwise/api.py

```diff
@@ -236,19 +234,26 @@ def _prepare_error_response(self, response):
                                                     error)
         return msg
 
+    @property
+    def api_base_url(self):
+        # This will be for this particular API-
+        # i.e. https://connectwise.example.com/v4_6_release/apis/3.0/service/
+        api_base_url, _ = self.build_api_base_url(False)
```

Comment:
I dont think that False has to be there, does it?

Reply:
I think it should because:

1. in the inner method, `def fetch_api_codebase(self, server_url, company_id, force_fetch=True):`, the `force_fetch` default value is True.
2. So, when this `api_base_url` is called at the first time, it will send remote request via line 125, `if not codebase_from_cache or force_fetch`
3. For this reason, it seems to be used w/ False tag before: 
![image](https://user-images.githubusercontent.com/73486314/140553449-c7c72254-b923-4fbb-901c-d5a19124c3ae.png)

Do I miss something by mistake? Then, please let me know.

Reply:
Oh yes, I apologize, I thought it was the reverse. Same question repeated then, but for line 343 where it specifies `True` and the default value is `True`. 

And can you add the `force_fetch=False` here on line 241 for readability for future devs?

Reply:
Considering all the above, it is done as follows:

![image](https://user-images.githubusercontent.com/73486314/140561044-1e7e213a-65d3-48be-a0e9-3c46e3aaddd3.png)

Thanks Sam